acto/runner/runner.py

```diff
@@ -311,10 +313,17 @@ def collect_operator_log(self) -> list:
             )
         else:
             logger.error("Failed to find operator pod")
-
-        log = self.core_v1_api.read_namespaced_pod_log(
-            name=operator_pod_list[0].metadata.name, namespace=self.namespace
-        )
+        if self.operator_container_name != None:
+            log = self.core_v1_api.read_namespaced_pod_log(
+            name=operator_pod_list[0].metadata.name, 
+            namespace=self.namespace,
+            container=self.operator_container_name
+            )
+        else: 
+            log = self.core_v1_api.read_namespaced_pod_log(
```

Comment:
@xinze-zheng Could you check if there are multiple containers here and add an error log if there are more than two containers?

Reply:
Done. Log and error if no target operator is specified and more than one container is detected.